cpan/Scalar-List-Utils/t/exotic_names.t

```diff
@@ -45,7 +45,7 @@ sub caller3_ok {
         ),
     );
 
-    $expected =~ s/'/::/g if $] < 5.037009 || $] >= 5.041_004;
+    $expected =~ s/'/::/g if $] < 5.041_003;
```

Comment:
I haven't been keeping up with when ' as separator was legal vs illegal. Is this change consistent with that, and preferred to what was here before?